include/attribute.h

```diff
@@ -28,6 +28,22 @@ struct BagWeapon
     u16 isGoodWeapon;
 }; /* size = 0x1C */
 
+struct GuardimalAttrib
+{
+    /*0x00*/ u8 guardimalType; // 00-04
+    /*0x01*/ u8 guardimalLevel;
+    /*0x02*/ u16 guardimalName[0x7]; // Something related to guardimal name
+    /*0x10*/ u16 guardimalHp;
+    /*0x12*/ u16 unk76;
```

Comment:
I'd say you should still use unk12 here. unk76 is very confusing when the struct is not directly accessed from the saveblock.